ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/files.yml

```diff
@@ -41,19 +41,19 @@ files:
   # --- Helm charts ---
   'https://charts.bitnami.com/bitnami/node-exporter-2.3.17.tgz':
     sha256: ec586fabb775a4f05510386899cf348391523c89ff5a1d4097b0592e675ade7f
-    deps: [kubernetes-master]
+    deps: [kubernetes-master, k8s-as-cloud-service]
 
   'https://helm.elastic.co/helm/filebeat/filebeat-7.12.1.tgz':
     sha256: 5838058fe06372390dc335900a7707109cc7287a84164ca245d395af1f9c0a79
-    deps: [kubernetes-master]
+    deps: [kubernetes-master, k8s-as-cloud-service]
 
   'https://charts.rook.io/release/rook-ceph-v1.8.8.tgz':
     sha256: f67e474dedffd4004f3a0b7b40112694a7f1c2b1a0048b03b3083d0a01e86b14
-    deps: [kubernetes-master]
+    deps: [kubernetes-master, k8s-as-cloud-service]
```

Comment:
I was asked yesterday about rook and k8s_as_cloud_service by @to-bar - and my response was that we never tested rook with k8s_as_cloud_service (no requirements to do so). Therefore I would say that we don't have to add `k8s-as-cloud-service` group for `rook-ceph` and `rook-ceph-cluster` charts.

Reply:
Maybe lets test if it works, if not remove?:D